variables.tf

```diff
@@ -1,24 +1,383 @@
-/********************************************************************
+# ##################################################
+# # Azure and IBM Authentication Variables
+# ##################################################
 
-This file is used to declare ROOT varaibles.
+variable "TF_VERSION" {
+  description = "terraform version"
+  type        = string
+  default     = "0.13"
+}
+
+variable "ibmcloud_api_key" {
```

Comment:
Can we remove declaration of providers at module and all its attribute..The example section should all these

Reply:
Removed provider block & variables from module